packages/android_intent/lib/android_intent.dart

```diff
@@ -138,6 +138,17 @@ class AndroidIntent {
     await _channel.invokeMethod<void>('launch', _buildArguments());
   }
 
+  /// Check whether the intent can be resolved to an activity.
+  ///
+  /// This works only on Android platforms.
+  Future<bool> canResolve() async {
```

Comment:
nit: Should this be `canResolveActivity`? Especially since there are other resolve types. I agree that we don't need to add them now, but I think it would make sense to name this just based off the specific method it's adding today instead of being more general.

Reply:
Yes, and the next change, if anyone ever needs it, could be `canResolveService` or `canResolveContentProvider`. But not for now. :)